paddlets/analysis/frequency_domain.py

```diff
@@ -43,7 +43,7 @@ class FFT(Analyzer):
         None
     """
 
-    def __init__(self, fs: float = 0, norm: bool = True, half: bool = True, **kwargs):
+    def __init__(self, fs: Optional[float] = 0, norm: Optional[bool] = True, half: Optional[bool] = True, **kwargs):
```

Comment:
should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional